packages/semantic-ui/src/SelectWidget/SelectWidget.js

```diff
@@ -94,10 +97,12 @@ function SelectWidget({
     // eslint-disable-next-line no-shadow
     target: { value },
   }) => onFocus && onFocus(id, processValue(schema, value));
+
   return (
     <Form.Dropdown
       key={id}
       name={name}
+      label={label || schema.title}
```

Comment:
Shouldn't we use getDisplayLabel here as well?

Reply:
I don't think so, as I have seen all other themes not do this? must be a reason why, unless you see otherwise I can do the same

Reply:
It's probably a mistake, I think all widgets should be using getDisplayLabel even in other themes. For now, though, let's just leave it as is for consistency.